v1/util/index.js

```diff
@@ -353,7 +355,9 @@ function flattenJson(data) {
       if (isEmpty && prop) result[prop] = {};
     }
   }
-  recurse(data, "");
+  if (data) {
```

Comment:
should we add the validation for primitive types as well, like 
flattenJson("abc")/ flattenJson([1,2,3]) etc?

Reply:
Done